src/d3a/d3a_core/export.py

```diff
@@ -204,7 +204,8 @@ def _export_area_offers_bids_csv_files(self, area, directory, file_suffix,
         (*-bids/offers/balancing-offers.csv files)
         return: dict[out_keys]
         """
-        file_path = self._file_path(directory, f"{area.slug}-{file_suffix}")
+        file_path = self._file_path(directory,
+                                    f"{area.slug}-{file_suffix}")
```

Comment:
There are a lot of these refatorings in this PR. Can you tell me the reason fo this?